hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -57,7 +57,18 @@ spec = do
           -- Using hex representation of aliceSk's HydraVerificationKey
           shouldRender "Party d5bf4a3fcce71"
           sendInputEvent $ EvKey (KChar 'q') []
-
+      it "report feedback for ever" $
+        \TUITest{sendInputEvent, shouldRender} -> do
+          threadDelay 1
+          shouldRender "connected"
+          shouldRender "Idle"
+          sendInputEvent $ EvKey (KChar 'f') []
+          threadDelay 1
+          shouldRender "Invalid command: Fanout"
+          threadDelay 1
+          shouldRender "Invalid command: Fanout"
+          threadDelay 1
+          shouldRender "Invalid command: Fanout"
```

Comment:
Was this test red on `master`?

If not, there is no point in having it. I rather have a test less than one which does not check what it intends to be checking.